src/status_im/contexts/wallet/common/collectibles_tab/view.cljs

```diff
@@ -48,17 +49,25 @@
         :image       (resources/get-themed-image :no-collectibles theme)}]
 
       :else
-      [rn/flat-list
-       {:data                     collectibles
-        :style                    {:flex 1}
-        :content-container-style  style/list-container-style
-        :window-size              11
-        :num-columns              2
-        :render-fn                (fn [item index]
-                                    (render-fn item
-                                               index
-                                               current-account-address
-                                               on-collectible-press
-                                               on-collectible-long-press))
-        :on-end-reached           on-end-reached
-        :on-end-reached-threshold 4}])))
+      ;; TODO:
```

Comment:
@ulisesmac - what to do here, should we create a follow up issue? 🤔

Reply:
I think so, we can create a follow-up to improve the code and performance, it's not creating problems so we can address it when we have more time :+1: